### PR TITLE
regression feature

### PR DESCRIPTION
**Pull request by lucasborin**

### CODE REVIEW

File: src/examples/y_demo_failures.clas.testclasses.abap [hunk 1]
```diff
@@ -1,56 +1,19 @@
...
-    CALL FUNCTION 'DATE_TO_DAY' IN UPDATE TASK
-      EXPORTING
-        date    = sy-datum
-      IMPORTING
-        weekday = date.
-
-    CALL FUNCTION 'DATE_TO_DAY' STARTING NEW TASK 'task' DESTINATION dest
-      EXPORTING
-        date = sy-datum.
-
```
Comment (lucasborin) [inline src/examples/y_demo_failures.clas.testclasses.abap:53]: I have simplified the external_call_in_ut to report it only once in the example class as the check behavior is being tested on its unit test.

File: src/foundation/y_code_pal_service.clas.abap [hunk 2]
```diff
@@ -124,4 +143,97 @@ CLASS y_code_pal_service IMPLEMENTATION.
...
+    IF result IS INITIAL.
+      RETURN.
+    ENDIF.
+    result = |Non-Executed Checks:<br>{ result }|.
+  ENDMETHOD.
+
+  METHOD execute_unit_test.
+    raise_internal_server_error( ).
+    RETURN.
+  ENDMETHOD.
```
Comment (lucasborin) [inline src/foundation/y_code_pal_service.clas.abap:237]: It will be implemented in the future.

File: src/foundation/y_code_pal_service.clas.abap [hunk 3]
```diff
@@ -124,4 +143,97 @@ CLASS y_code_pal_service IMPLEMENTATION.
...
+
+  METHOD list_non_executed_checks.
+    result = checks.
+    LOOP AT findings ASSIGNING FIELD-SYMBOL(<finding>).
+      DELETE result WHERE checkid = <finding>-test.
+    ENDLOOP.
+    " not supported
+    DELETE result WHERE checkid = 'Y_CHECK_PROFILE_MESSAGE'
+                     OR checkid = 'Y_CHECK_TEST_SEAM_USAGE'
+                     OR checkid = 'Y_CHECK_FUNCTION'.
```
Comment (lucasborin) [inline src/foundation/y_code_pal_service.clas.abap:220]: It is not part of the Y_DEMO_FAILURES yet.

File: src/examples/y_demo_failures.clas.testclasses.abap (hunk 1 above)
Comment (lucasborin) [reply src/examples/y_demo_failures.clas.testclasses.abap:53]: By the way, this UT was failing.

**Review by estevao-schultz-neto-SAP — APPROVED** [review]

**Pull request merged**